include/mbedtls/ssl.h

```diff
@@ -3452,7 +3326,7 @@ int mbedtls_ssl_conf_own_cert( mbedtls_ssl_config *conf,
  * \returns A negative error code on failure.
  */
 
-int mbedtls_ssl_conf_tls13_key_exchange( mbedtls_ssl_config* conf,
+int mbedtls_ssl_conf_ke( mbedtls_ssl_config* conf,
```

Comment:
Why was the renaming `mbedtls_ssl_conf_ke() -> mbedtls_ssl_conf_tls13_key_exchange()` reverted here?

Reply:
I checked the code and noticed that this change was unnecessary. I reverted it back.